M2Mqtt/MqttClient.cs

```diff
@@ -1388,20 +1388,20 @@ private void ReceiveThread()
         /// </summary>
         private void KeepAliveThread()
         {
-            int delta = 0;
-            int wait = _keepAlivePeriod;
```

Comment:
you will cast it to an int later on, so not sure why you are changing it. I don't see any reason.